packer/http/centos-7.2/ks.cfg

```diff
@@ -0,0 +1,80 @@
+install
```

Comment:
Since we are not adding non openstack or ganeti images I guess this file can be removed?